src/gsi/read_seviri.f90

```diff
@@ -286,10 +284,10 @@ subroutine read_seviri(mype,val_sev,ithin,rmesh,jsatid,&
 !       Read through each record
         call ufbint(lnbufr,hdr,nhdr,1,iret,hdrsevi)
         if(nint(hdr(1)) /= kidsat) cycle read_loop
-        if (clrsky) then     ! asr bufr has no sza
+!       if (clrsky) then     ! asr bufr has no sza, asr bufr has sza since 2017.07
 !          remove the obs whose satellite zenith angles larger than 65 degree
            if ( hdr(ilzah) > r65 ) cycle read_loop
-        end if
+!       end if
```

Comment:
Asked Haixia for guidance.  She would like to leave the if and end if comments in the code to remind developers that there is no SZA in the bufr file before 2017.07.  She said the line "remove the obs whose satellite zenith angles larger than 65 degree" may be removed.  Her guidance followed.

Since if-then block commented out, adjust indentation of line 288 to correspond to adjacent lines.